src/components/ArticleContainer.jsx

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect} from 'react'
+import Spinner from 'react-bootstrap/Spinner';
+import axios from 'axios'
+import ArticleCard from './ArticleCard';
+import Row from 'react-bootstrap/Row';import Col from 'react-bootstrap/Col';
+
+
+
+function ArticleContainer() {
+    const [articles, setArticles] = useState([]);
+    const [loading, setLoading] = useState(true);
+
+    useEffect(() => {
+        axios.get('https://nc-news-2.onrender.com/api/articles')
```

Comment:
It would be great if you could extract your axios requests to a utils file so you can have them all in one place. You would then export/import each function as needed. 

If you have your axios requests in one file you could then use axios.create to set up a base URL for your app

Reply:
Hi Hannah, thanks will do